packages/office-ui-fabric-react/src/components/Icon/examples/Icon.Color.Example.tsx

```diff
@@ -1,17 +1,35 @@
-/* tslint:disable:no-unused-variable */
-import * as React from 'react';
-/* tslint:enable:no-unused-variable */
+// tslint:disable:jsx-no-lambda
 
+import * as React from 'react';
 import { Icon } from 'office-ui-fabric-react/lib/Icon';
-import './IconExample.scss';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+const IconExampleClass = mergeStyles({
+  fontSize: 50,
+  height: 50,
+  width: 50,
+  margin: '0 25px'
+});
 
 export class IconColorExample extends React.Component<any, any> {
   public render() {
     return (
       <div>
-        <Icon iconName={ 'CompassNW' } className={ 'ms-IconExample ms-IconColorExample-deepSkyBlue' } />
-        <Icon iconName={ 'Dictionary' } className={ 'ms-IconExample ms-IconColorExample-greenYellow' } />
-        <Icon iconName={ 'TrainSolid' } className={ 'ms-IconExample ms-IconColorExample-salmon' } />
+        <Icon
+          iconName={ 'CompassNW' }
+          className={ IconExampleClass }
+          getStyles={ () => ({ root: { color: 'deepskyblue' } }) }
+        />
+        <Icon
+          iconName={ 'Dictionary' }
+          className={ IconExampleClass }
+          getStyles={ () => ({ root: { color: 'greenyellow' } }) }
```

Comment:
lamdas in a high usage component like icon are probably a bad idea. Should we update this to use a named function instead so people know to follow the best practice?

Reply:
good point!